etl/tasks.py

```diff
@@ -14,3 +17,24 @@ def import_file_task(
         is_background=True,
         is_remote=True,
     )
+
+
+@shared_task
+def remove_file_task(files: []):
+    account = None
+    for f in files:
```

Comment:
I might suggest something like ``for f in File.objects.filter(pk__in=files)`` here. Then all that needs to happen in the for loop is:
* Log the message
* ``f.delete()``